README.md

```diff
@@ -24,12 +24,13 @@ Starting HDFS-Mesos
 
 Using HDFS
 --------------------------
-See some of the many HDFS tutorials out there for more details and explore the web UI at <br>`http://<ActiveNameNode>:50070`.</br>
+See some of the many HDFS tutorials out there for more details and explore the web UI at <br>`http://<ActiveNameNode>:50070`.</br> 
+Note that you can access commands through `hdfs://<ConfigurableFrameworkName>/` where hdfs is the default framework name. 
```

Comment:
Go ahead and use the configuration key for this, like "`hdfs://<mesos.hdfs.framework.name>/` (default: `hdfs://hdfs/`)"